README.adoc

```diff
@@ -269,6 +274,78 @@ properties are automatically stored in the inventory. Go back to http://localhos
 you see a new entry for `[system-ip-address]`. 
 
 
+== Externalizing server configuration
+
+// File 0
+inventory/server.xml
+[source, xml, linenums, indent=0, role="code_column"]
+----
+include::finish/inventory/src/main/liberty/config/server.xml[]
+----
+
+As mentioned in the beginning of this guide, one of the advantages of using
+containers is that they are portable and can be moved and deployed efficiently
+across all of your DevOps environments. Configuration often changes across
+different environments, and by externalizing your server's configuration, you
+can simplify the development process.
+
+Imagine a scenario where you are developing an Open Liberty application on
```

Comment:
Change,
"Imagine a scenario where you are developing an Open Liberty application on
port `9081` but when it comes time to deploy it to production, it must be available
on port `9091`."
to,
"Imagine a scenario where you are developing an Open Liberty application on
port `9081` but to deploy it to production, it must be available
on port `9091`."